### ISSUE TITLE
EnvironmentDetector using undefined method

### ISSUE THREAD

**Issue by thafner** [acquia/drupal-recommended-settings#39]
**Describe the bug**
The `getSiteName` method of the EnvironmentDetector class calls `drs_config->get()`, but drs_config is a ConfigInitializer object, which does not have a `get()` method.

```Call to undefined method Acquia\Drupal\RecommendedSettings\Config\ConfigInitializer::get()```

**To Reproduce**
Set up DRS locally and attempt to load and ACSF multisite locally.  hard error occurs
```Call to undefined method Acquia\Drupal\RecommendedSettings\Config\ConfigInitializer::get()```

**Notes**
The issue seems to revolve around creation of the ConfigInitializer object.  when the `initialize()` method [is called](https://github.com/acquia/drupal-recommended-settings/blob/83c2be59949c482c9c829032fa66f45f9b5c85bd/src/Config/ConfigInitializer.php#L85), it returns an instance of the type `ConfigInitializer` rather than `DefaultConfig`.  BLT handled this by [returning an instance](https://github.com/acquia/blt/blob/0a2c6da746c2b59ab3569cb2c41b021655b1f3bd/src/Robo/Config/ConfigInitializer.php#L128) of `DefaultConfig`.  The ConfigInitializer class does not have a `get()` method, unlike the `DefaultConfig` class which does.

**Comment by sdjili** [comment]
Same issue after trying to replace blt with drs like describe here 
https://dev.acquia.com/tutorial/you-dont-need-blt-acquia-cloud

**Comment by aneek** [comment]
+1.
Same issue with ACSF, DDEV

**Comment by vishalkhode1** [comment]
This is fixed in develop branch and planned for release this week. Thanks,

**Comment by aneek** [comment]
@vishalkhode1 - thanks. Now via UI a site can be installed properly. However, `drush` commands are failing.

```
PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function Acquia\Drupal\RecommendedSettings\Drush\Commands\MultisiteDrushCommands::__construct(), 0 passed and exactly 1 expected in /var/www/html/vendor/acquia/drupal-recommended-settings/src/Drush/Commands/MultisiteDrushCommands.php:34
Stack trace:
#0 [internal function]: Acquia\Drupal\RecommendedSettings\Drush\Commands\MultisiteDrushCommands->__construct()
#1 /var/www/html/vendor/league/container/src/Definition/Definition.php(225): ReflectionClass->newInstanceArgs(Array)
#2 /var/www/html/vendor/league/container/src/Definition/Definition.php(181): League\Container\Definition\Definition->resolveClass('Acquia\\Drupal\\R...')
#3 /var/www/html/vendor/league/container/src/Definition/Definition.php(160): League\Container\Definition\Definition->resolveNew()
#4 /var/www/html/vendor/league/container/src/Definition/DefinitionAggregate.php(79): League\Container\Definition\Definition->resolve()
#5 /var/www/html/vendor/league/container/src/Container.php(175): League\Container\Definition\DefinitionAggregate->resolve('Acquia\\Drupal\\R...')
#6 /var/www/html/vendor/league/container/src/Container.php(118): League\Container\Container->resolve('Acquia\\Drupal\\R...')
#7 /var/www/html/vendor/consolidation/robo/src/Robo.php(587): League\Container\Container->get('Acquia\\Drupal\\R...')
#8 /var/www/html/vendor/consolidation/robo/src/Robo.php(542): Robo\Robo::instantiate('Acquia\\Drupal\\R...')
#9 /var/www/html/vendor/consolidation/robo/src/Robo.php(529): Robo\Robo::registerSingle(Object(Drush\Application), 'Acquia\\Drupal\\R...')
#10 /var/www/html/vendor/consolidation/robo/src/Runner.php(319): Robo\Robo::register(Object(Drush\Application), Array)
#11 /var/www/html/vendor/drush/drush/src/Application.php(353): Robo\Runner->registerCommandClasses(Object(Drush\Application), Array)
#12 /var/www/html/vendor/drush/drush/src/Runtime/Runtime.php(119): Drush\Application->configureAndRegisterCommands(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput), Array, Object(Composer\Autoload\ClassLoader))
#13 /var/www/html/vendor/drush/drush/src/Runtime/Runtime.php(51): Drush\Runtime\Runtime->doRun(Array, Object(Symfony\Component\Console\Output\ConsoleOutput))
#14 /var/www/html/vendor/drush/drush/drush.php(79): Drush\Runtime\Runtime->run(Array)
#15 /var/www/html/vendor/drush/drush/drush(4): require('/var/www/html/v...')
#16 /var/www/html/vendor/bin/drush(119): include('/var/www/html/v...')
#17 {main}
  thrown in /var/www/html/vendor/acquia/drupal-recommended-settings/src/Drush/Commands/MultisiteDrushCommands.php on line 34

Fatal error: Uncaught ArgumentCountError: Too few arguments to function Acquia\Drupal\RecommendedSettings\Drush\Commands\MultisiteDrushCommands::__construct(), 0 passed and exactly 1 expected in /var/www/html/vendor/acquia/drupal-recommended-settings/src/Drush/Commands/MultisiteDrushCommands.php:34
Stack trace:
#0 [internal function]: Acquia\Drupal\RecommendedSettings\Drush\Commands\MultisiteDrushCommands->__construct()
#1 /var/www/html/vendor/league/container/src/Definition/Definition.php(225): ReflectionClass->newInstanceArgs(Array)
#2 /var/www/html/vendor/league/container/src/Definition/Definition.php(181): League\Container\Definition\Definition->resolveClass('Acquia\\Drupal\\R...')
#3 /var/www/html/vendor/league/container/src/Definition/Definition.php(160): League\Container\Definition\Definition->resolveNew()
#4 /var/www/html/vendor/league/container/src/Definition/DefinitionAggregate.php(79): League\Container\Definition\Definition->resolve()
#5 /var/www/html/vendor/league/container/src/Container.php(175): League\Container\Definition\DefinitionAggregate->resolve('Acquia\\Drupal\\R...')
#6 /var/www/html/vendor/league/container/src/Container.php(118): League\Container\Container->resolve('Acquia\\Drupal\\R...')
#7 /var/www/html/vendor/consolidation/robo/src/Robo.php(587): League\Container\Container->get('Acquia\\Drupal\\R...')
#8 /var/www/html/vendor/consolidation/robo/src/Robo.php(542): Robo\Robo::instantiate('Acquia\\Drupal\\R...')
#9 /var/www/html/vendor/consolidation/robo/src/Robo.php(529): Robo\Robo::registerSingle(Object(Drush\Application), 'Acquia\\Drupal\\R...')
#10 /var/www/html/vendor/consolidation/robo/src/Runner.php(319): Robo\Robo::register(Object(Drush\Application), Array)
#11 /var/www/html/vendor/drush/drush/src/Application.php(353): Robo\Runner->registerCommandClasses(Object(Drush\Application), Array)
#12 /var/www/html/vendor/drush/drush/src/Runtime/Runtime.php(119): Drush\Application->configureAndRegisterCommands(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput), Array, Object(Composer\Autoload\ClassLoader))
#13 /var/www/html/vendor/drush/drush/src/Runtime/Runtime.php(51): Drush\Runtime\Runtime->doRun(Array, Object(Symfony\Component\Console\Output\ConsoleOutput))
#14 /var/www/html/vendor/drush/drush/drush.php(79): Drush\Runtime\Runtime->run(Array)
#15 /var/www/html/vendor/drush/drush/drush(4): require('/var/www/html/v...')
#16 /var/www/html/vendor/bin/drush(119): include('/var/www/html/v...')
#17 {main}
  thrown in /var/www/html/vendor/acquia/drupal-recommended-settings/src/Drush/Commands/MultisiteDrushCommands.php on line 34
 [warning] Drush command terminated abnormally.
Failed to run drush : exit status 1
```

Drush version: 11.6.0
Drupal version: 10.1.x (Multi-site ACSF)

**Comment by vishalkhode1** [comment]
Fixed and released in [1.0.0](https://github.com/acquia/drupal-recommended-settings/releases/tag/1.0.0).
